docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -88,7 +85,41 @@ gradient boosting methodologies.
 //end::regression-algorithms[]
 
 
-[discrete]
+[[dfa-regression-lossfunction]]
+===== Loss functions for {regression} analyses
+
+Loss function is for measuring how well a given {ml} model fits the specific 
+data set. The loss function boils down all the different aspects of the model to 
+a single number. The bigger the difference between the prediction and the ground 
+truth, the higher the value of the loss function. Loss functions are used 
+automatically in the background during 
+<<hyperparameters,hyperparameter optimization>> and when training the decision 
+trees to compare the various iterations of the model and decide which iteration 
+performs better. 
+
+In the {stack}, there are three different types of loss function:
+
+* https://en.wikipedia.org/wiki/Mean_squared_error[mean squared error (`mse`)]
+* mean squared logarithmic error (`msle`; a variation of `mse`)
+* https://en.wikipedia.org/wiki/Huber_loss#Pseudo-Huber_loss_function[Pseudo-Huber loss (`huber`)]
```

Comment:
Can we add formulas here? Otherwise, it is unclear what `t` and `delta` mean.

Reply:
I would also like to add a sentence to the purpose of each loss function. Something like 
MSE: The default choice if no additional information about the data set is available.
MSLE: If you know that the target values are all positive with a long tail distribution (e.g. prices, population, etc.)
Huber: If you want to prevent that the model tries to fit the outliers at the cost of fitting regular data.